WebDriverAgentLib/Categories/XCUIElement+FBFind.m

```diff
@@ -13,21 +13,32 @@
 #import "FBElementTypeTransformer.h"
 #import "XCElementSnapshot.h"
 #import "XCElementSnapshot+FBHelpers.h"
+#import "XCUIElement+FBUtilities.h"
 #import "XCUIElement+FBWebDriverAttributes.h"
 
 @implementation XCUIElement (FBFind)
 
 
 #pragma mark - Search by ClassName
 
-- (NSArray<XCUIElement *> *)fb_descendantsMatchingClassName:(NSString *)className
+- (NSArray<XCUIElement *> *)fb_descendantsMatchingClassName:(NSString *)className shouldReturnAfterFirstMatch:(BOOL)shouldReturnAfterFirstMatch
```

Comment:
I don't thin query is resolving elements, but If you measured it, it would be pointless for me to argue with you :).